infoportal-server/src/core/EmailService.ts

```diff
@@ -0,0 +1,34 @@
+import {logger} from '../helper/Logger'
+import {GlobalEvent} from './GlobalEvent'
+import sendEmail from './EmailHelper'
+
+const log = logger('EmailService');
+
+class EmailService {
+  constructor(private event = GlobalEvent.Class.getInstance()) {
+    this.initializeListeners();
+  }
+
+  private initializeListeners() {
+    this.event.listen(GlobalEvent.Event.KOBO_TAG_EDITED, this.handleTagEdited);
+  }
+
+  private handleTagEdited = async (params: GlobalEvent.KoboTagEditedParams) => {
```

Comment:
This you are writting a mail service, it's more human logical to write a method `sendCfmNotification` method and to do the `if` condition in the event listener above.